.travis.yml

```diff
@@ -165,15 +165,27 @@ jobs:
           -in astronomer-prod-fec60ad8e5e2.json.enc \
           -out astronomer-prod-fec60ad8e5e2.json -d
       deploy:
-        provider: gcs
-        edge: true
-        bucket: "pip.astronomer.io"
-        acl: public-read
-        local_dir: ./dist
-        upload_dir: simple
-        key_file: astronomer-prod-fec60ad8e5e2.json
-        on:
-          all_branches: true
+        - provider: gcs
+          edge: true
+          bucket: "pip.astronomer.io"
+          acl: public-read
+          local_dir: ./dist
+          upload_dir: simple
+          key_file: astronomer-prod-fec60ad8e5e2.json
+          on:
+            all_branches: true
+        - provider: gcs
+          edge: true
+          bucket: "pip.astronomer.io"
+          acl: public-read
+          local_dir: ./
```

Comment:
I think this will upload the file as pip.astronomer.io/simple/dist/astronomer-certified/*.build

Reply:
Why do you think so?

I looked at the following examples:
- https://docs.travis-ci.com/user/deployment/s3/#deploy-from-only-one-folder
- https://docs.travis-ci.com/user/deployment/s3/#deploy-to-a-specific-s3-folder
- https://stackoverflow.com/a/28579635/5691525

Can we try if it does work, I will fix it today itself if it doesn't work?

Reply:
Because before we uploaded `./dist`, which contained `./dist/apache-airflow/apache-airflow-X.whl`, and that ended up as `simple/apache-airflow/apache-airflow-X.whl`.

So uploading `./` containing `./dist/apache-airflow/x.build` will be uploaded as `simple/dist/apache-airflow/x.build`

Reply:
But the `x.build` file is not stored in `./dist` folder, it is just stored in `./`:

```
echo "${CURRENT_AC_VERSION}" > astronomer-certified/latest-"$AIRFLOW_BASE_VESION".build

```

Reply:
aah, I need to update my example above the command

Reply:
Updated the example